README.md

```diff
@@ -2,6 +2,11 @@
 
 A tool for creating an accessible color palette which corresponds to the [WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) and [WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html) criteria for color contrast
 
+## Screenshot
+
+<img width="736" alt="image" src="https://user-images.githubusercontent.com/83456083/176761271-e6971a39-a2e8-407f-b00f-78834efa9799.png">
```

Comment:
Can you please make the alt text descriptive so that screen reader users know what this is an image of. something like "Screenshot of website showing 3 color inputs and the possible color combinations"

Then this should be ready to merge :)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Added